website/templates/base.html

```diff
@@ -16,29 +16,40 @@
 
 <body>
     <div class="navbar">
-        <nav>
-            <a href="{% url 'home' %}">Home</a>
-            <a href="{% url 'advanced_search' %}">Recherche avancée</a>
-            {% if user.is_authenticated %}
-                {% if user.is_staff %}
-                <a href={% url 'add_game' %}>Ajouter un jeu</a>
-                <a href="/admin/">Admin</a>
-                {% endif %}
-                <a href={% url 'logout' %}>Se déconnecter</a>
-            {% else %}
-                <a href="{% url 'login' %}">Se connecter</a>
-                <a href="{% url 'signup' %}">S'inscrire</a>
-            {% endif %}
-
         {% if simple_search %}
-        <form method="post" action="{% url 'advanced_search' %}">
-            {% csrf_token %}
-            {{ simple_search.as_p }}
-            <button type="submit">Rechercher</button>
+        <form class="simple_search" method="post" action="{% url 'advanced_search' %}">
+            <div class="row">
+                <div class="col-10 my-auto">
+                    {% csrf_token %}
+                    {{ simple_search.game_name }}
+                </div>
+                <div class="col-2">
+                    <button class="button_search btn btn-danger" type="submit">Rechercher</button>
+                </div>
+            </div>
         </form>
         {% endif %}
-    </nav>
- </div>
+        <div class="dropdown ps-3 ms-2"></div>
```

Comment:
```suggestion
        <div class="dropdown ps-3 ms-2">
```
Il y a un </div> en trop, je sais pas si c'est celui là, mais du coup ça centre le menu...